### ISSUE TITLE
t2a unreachable

### ISSUE THREAD

**Issue by edolstra** [NixOS/nixos-org-configurations#127]
```
Oct 20 13:18:21 ceres hydra-queue-runner[10314]: possibly transient failure building ‘/nix/store/vlpwymfgjw6ankg906aqkjfsb30yk41a-nixpkgs-metrics.drv’ on ‘root@t2a.cunat.cz’: error: --- Error --- hydra-queue-runner
Oct 20 13:18:21 ceres hydra-queue-runner[10314]: cannot connect to ‘root@t2a.cunat.cz’: ssh: connect to host t2a.cunat.cz port 22: No route to host
```

**Comment by cole-h** [comment]
This appears to be blocking `nixos-unstable` from advancing (and maybe some other "large" channels) due to being the only system with the `benchmark` system feature as required by the `nixpkgs-metrics` job: https://github.com/NixOS/nixpkgs/blob/4b3e97daa4869541938b4dad6448dc660625fa21/pkgs/top-level/metrics.nix#L7 (https://hydra.nixos.org/eval/1620632#tabs-unfinished).

Relatedly, t2b also appears to be unreachable as well: https://hydra.nixos.org/build/128835041.

**Comment by edolstra** [comment]
I've cancelled it manually. Hydra is supposed to cancel builds with an unsupported machine type automatically...

**Comment by cole-h** [comment]
Cross-referencing here:

> I believe you can re-enable `t2a` again.  (I'm not sure who disabled them this time.  There's no `t4b` yet but I expect to revive it eventually, no ETA though.)

_Originally posted by @vcunat in https://github.com/NixOS/nixos-org-configurations/issues/118#issuecomment-717546260_

**Comment by vcunat** [comment]
Oh, I haven't noticed this thread.  (Though I did know about the disconnection all the time.)

**Comment by edolstra** [comment]
@vcunat Thanks, I've re-enabled it.

**Comment by vcunat** [comment]
Uh, apparently 8G RAM does not suffices for the `metrics` job anymore.

Until May 2019 4G was enough.  The job expression remained the same, it's only the packages that are growing.  We've more than doubled the requirements in one year and a half.  It may not seem nice but operations passing over all of them should just be avoided.

I believe I can use RAM from the dead `t4a` and get `t2a` up to 12G, some time later this week.

**Comment by vcunat** [comment]
`t4b` is up now.  And now that I got rid of DHCPv6, this kind of disconnection should never happen anymore 📈

**Comment by edolstra** [comment]
@vcunat Thanks!

**Comment by vcunat** [comment]
> I _think_ I can use RAM from the dead

No, I confirmed my suspicion that this old machine won't accept the RAM, so I don't think I'll ever be able to get this one above 8G.  For now I disabled `tmpOnTmpfs`, which might add noticeable jitter to timings but that's better than failures.

The drive is 7.2k 2.5" ext4; I guess I could buy some super-cheap SSD in case that seemed important, but I fear we'll exhaust the 8G within a year anyway... perhaps I could switch `t4b` to this benchmark-only role by that point (it has 16G).
